docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -521,6 +521,8 @@ By default, variables are merged/flattened to the specific host before a play is
 - child group
 - host
 
+.. note:: See :ref:`ansible_variable_precedence` for information about merging variables across different sources (for example, `group_vars` in parent groups override inventory variables in child groups).
```

Comment:
I think this should go after line 526 and be a little more clear in the importance to users.

"Ansible merges variables from different sources and applies precedence to some variables over others according to a set of rules. For example, variables that occur higher in an inventory can override variables that occur lower in the inventory. See :ref:`ansible_variable_precedence` for more information."

Reply:
I slightly modified your suggestiion and applied that. "Lower and higher in inventory" seemed a little unclear to me, so i tried to clarify it.